[WIP] Updated section on reviewing an analysis report (#162)
migtools/mta-documentation: docs/topics/installing-mta-disconnected-environment.adoc

[id="installing-mta-disconnected-environment_{context}"]
= Installing {ProductShortName} on a disconnected environment

> When your system is in a disconnected environment, you can install the Migration Toolkit for Applications (MTA) command-line interface (CLI) by using an external computer to download the required images and by copying these images to the system you want to install MTA CLI on.

Reviewer: ```suggestion
When your system is in a disconnected environment, you can do the following to install the MTA CLI:
* Download the Migration Toolkit for Applications (MTA) command-line interface (CLI) images by using an external computer.
* Copy the downloaded images to the system you want to install the MTA CLI on.
```
This is just a suggestion :)
